### PR TITLE
lynis: update to 3.0.7.

### PR DESCRIPTION
**Pull request by jhe2**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Review by paper42 — COMMENTED** [review]
* add `$changelog`
* all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?

**Comment by jhe2** [comment]
> * add `$changelog`
> 
>     * all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?

I have added the changelog URL and removed the ``INSTALL``, ``README``  and ``CHANGELOG.md`` files from ``/usr/share/doc/lynis``, but kept ``FAQ``.

I've noticed that other packages sometimes also contain CHANGELOG files in their ``/usr/share/doc`` directories. Is it generally discouraged to include them there?

File: srcpkgs/lynis/template [hunk 1]
```diff
@@ -19,9 +20,6 @@ do_install() {
 	vbin lynis
 	vman lynis.8
 	vmkdir usr/share/doc/lynis
```
Comment (paper42) [inline srcpkgs/lynis/template:22]:
```suggestion
```

File: srcpkgs/lynis/template [hunk 2]
```diff
@@ -19,9 +20,6 @@ do_install() {
...
-	vinstall CHANGELOG.md 0644 usr/share/doc/lynis
 	vinstall FAQ 0644 usr/share/doc/lynis
```
Comment (paper42) [inline srcpkgs/lynis/template:23]:
```suggestion
	vdoc FAQ
```

**Comment by paper42** [comment]
> > * add `$changelog`
> >   
> >   * all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?
> 
> I have added the changelog URL and removed the `INSTALL`, `README` and `CHANGELOG.md` files from `/usr/share/doc/lynis`, but kept `FAQ`.
> 
> I've noticed that other packages sometimes also contain CHANGELOG files in their `/usr/share/doc` directories. Is it generally discouraged to include them there?

From what I can see, CHANGELOG is explicitly included only in 4 packages (durden fingerprint-gui go-ipfs hunspell-it_IT). I don't think it's necessary to include a changelog, it doesn't add any value to the documentation and people can check it online if they really want to.

**Comment by jhe2** [comment]
> From what I can see, CHANGELOG is explicitly included only in 4 packages (durden fingerprint-gui go-ipfs hunspell-it_IT). I don't think it's necessary to include a changelog, it doesn't add any value to the documentation and people can check it online if they really want to.

Of the packages installed on my system, the ``inxi`` package also comes with a changelog, so does ``flac`` (although in case of flac it appears to be installed implicitly).
I actually like not having to look online for things like that, but I'm all for consistency, so if such files are not supposed to be included, that's fine. I guess it is not of much use, if only some packages include a CHANGELOG files and other don't.

**Pull request merged**